src/main/java/com/powsybl/openloadflow/ac/equations/AcEquationSystemCreator.java

```diff
@@ -869,6 +933,15 @@ private static void createHvdcAcEmulationEquations(LfHvdc hvdc, EquationSystem<A
 
     private void createImpedantBranchEquations(LfBranch branch,
                                                EquationSystem<AcVariableType, AcEquationType> equationSystem) {
+        branch.getAdditionalOpenP1().clear();
```

Comment:
Is it really needed?

Reply:
Yes in case with the same LfNetwork we create several time the equation system